camp_1/README.md

```diff
@@ -904,39 +923,247 @@ func constructor{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr
+
+1. Open your Argent wallet and click on the settings icon at the top right.
+2. Select “Developer settings”.
+3. Select “Smart contract development”.
+4. Declare your contract to get the clash hash: select "Declare smart contract".
+5. Select “Click to upload contract JSON” to upload the JSON ABI, `voting_compiled.json`, obtained from compiling our contract. Then select the network the contract will be deployed on and the account used for deployment (who pays for the deployment?). Sign the transaction.
+
+<div align="center">
+    <img src="../misc/argent1.png">
+</div>
+
+6. We get redirected to the page with our contract’s class hash. In this case, our class hash is: `0x0679eec449818394722a31ecb38594343b2c687efab5404a3f75fd45d4c1fa0c`.
+
+<div align="center">
+    <img src="../misc/argent2.png">
+</div>
+
+7. To deploy our contract, we select the "Deploy smart contract" option in the "Smart contract deployment" section. We need to input our constructor's class hash and call data. 
+
+The value for salt can be any number you want; it’s there only to introduce randomness into the generated address for your soon-to-be-deployed smart contract.
+
+The unique field, combined with salt, can get the same address on different networks. For example, if we deploy to Goerli, passing the value 5 for salt, and do not check the unique address field, the smart contract will be deployed to a specific address; let’s say it’s 0xabc… We can now repeat the deployment process, but this time on Mainnet, and if we pass the same values for salt and the unique field, the smart contract will be deployed to the same address 0xabc… but this time on Mainnet. In this example, we will uncheck the unique field and write 0 in salt for simplicity, not because we want to preserve the address.|
+
+Unfortunately, until December 2022, Argent still needs to add the feature to deal with arrays in their UI deployer. This will change soon, but we can not deploy our voting contract using Argent's UI yet. We will deploy it using the `starknet cli` in the section. 
+
+8. Sign the transaction, and you’ll be redirected to a page with your deployed contract address 
+
+Our contract is now on-chain. We can interact with it using block explorers such as [StarkScan](https://testnet.starkscan.co/) or [Voyager](https://goerli.voyager.online/). These explorers have plenty of functionality. Some of the most used sections are the `read` and `write` tabs of a contract; here, you can interact with the `view` and `external` functions in your contract.
+
+
+### Deploying II: Open Zeppelin's Universal Deployer Contract (optional)
+
+This section will explain what is beneath the Argent's UI. Do this section if you want to get deeper into what deploying a contract in StarkNet means. This will help you later work with more complex contracts, e.g. deploying your customize account contracts. However, for most contracts, using Argent's UI will be enough.
```

Comment:
I would use "underneath" rather than "beneath" because the latter could be understood as something negative